docs/protocol.md

```diff
@@ -51,10 +51,16 @@ The following lists the parameters used by this version of the Sidetree protocol
 
 ## Sidetree Operations
 
-A [_DID Document_](https://w3c-ccg.github.io/did-spec/#ex-2-minimal-self-managed-did-document
-) is a document containing information about a DID, such as the public keys of the DID owner and service endpoints used. Sidetree protocol enables the creation of, lookup for, and updates to DID Documents through _Sidetree operations_.
+Sidetree protocol allows the following operations to be performed against a DID:
 
-An update operation to a document contains only the changes from the previous version of the document. Create and recover operations require a full document state of the DID as input.
+1. Create
+1. Resolve
+1. Update
+1. Recover
+1. Revoke
```

Comment:
`Deactivate`?

Reply:
I'll leave it as is and let #321 handle it. I want the doc to match the code, and plus we will need to remove that portion of the doc and refer to the new spec document going forward.